apollo-federation/tests/query_plan/mod.rs

```diff
@@ -1,5 +1,3 @@
 #[macro_use]
 mod build_query_plan_support;
 mod build_query_plan_tests;
-// TODO: port query-planner-js/src/__tests__/buildPlan.*.test.ts as new modules here
-mod operation_validations_tests;
```

Comment:
NOTE: this change is unrelated to the defer cleanup. Those tests were not implemented and their JS counterpart were covering general GraphQL validations that are already handled by `apollo-rs` and reuse fragment logic.